modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -38,6 +38,10 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * Google Cloud Platform (GCP): `LoadBalancerService` (with External scope)
 * Bare metal: `NodePortService`
 * Other: `HostNetwork`
+
+On GCP, AWS, and Azure you can configure the following `endpointPublishingStrategy` fields:
+* `loadBalancer.scope`
+* `loadBalancer.allowedSourceRanges`
 +
 [NOTE]
```

Comment:
I think this note probably belonged after the above "Other: `HostNetwork`" line, before your note

Reply:
Addressed